app/Http/Controllers/Auth/AuthController.php

```diff
@@ -119,16 +119,16 @@ public function register(RegisterRequest $registerRequest)
         $user = $this->userRepository->create(Arr::except($data, ['domain']));
 
         if ($user) {
-            $invited_users = $this->invitedTeamUserRepository->searchByEmail($data['email']);
-            if ($invited_users) {
-                foreach ($invited_users as $invited_user) {
-                    $team = $this->teamRepository->find($invited_user->team_id);
-                    if ($team) {
-                        $team->users()->attach($user, ['role' => 'collaborator', 'token' => $invited_user->token]);
-                        $this->invitedTeamUserRepository->delete($data['email']);
-                    }
-                }
-            }
+            // $invited_users = $this->invitedTeamUserRepository->searchByEmail($data['email']);
```

Comment:
do we need this in future?
if not then remove.

Reply:
yes, may be we will use. so for now ignore this commented code until we finalize the teams module.